### ISSUE TITLE
Change name "Child Groups" to "Sub Spaces"

### ISSUE THREAD

**Issue by gdecuir777** [dcavins/hierarchical-groups-for-bp#5]
Sorry if this is not the right place to ask for help. I'm new to GitHub and a rookie to programming. I'm trying  to simply change the name of the button label under the parent group from "Child groups" to "Sub spaces". I'm hoping this is a simple task to do. Anyone's help would be greatly appreciated.

**Comment by dcavins** [comment]
Hello, you can add a custom language file to change strings everywhere. See this article for a how-to:
https://codex.buddypress.org/getting-started/customizing/customizing-labels-messages-and-urls/

When completed, put the new file in this location on your server for it to be read:
`wp-content/languages/hierarchical-groups-for-bp/`, with the filenames `hierarchical-groups-for-bp-en_US.mo` for US English, for example.

Please let me know how the plugin is working for you. I haven't yet released it to the WP.org plugin repo and would appreciate the feedback.

**Comment by HiveMindNet** [comment]
Hi David - I'm not noticing any issues as such but just noticed this warning in the apache logs - any thoughts?

PHP Warning:  Missing argument 2 for HGBP_Public::make_permalink_hierarchical() in /var/www/WordpressMultisite/wp-content/plugins/hierarchical-groups-for-bp-master/public/class-hgbp.php on line 341

**Comment by gdecuir777** [comment]
no issues I'm having

**Comment by dcavins** [comment]
@gdecuir777 Were you able to make the label change you needed to make? If so, let's close this issue. Thanks!

**Comment by dcavins** [comment]
Hi @gdecuir777-

I've added a couple of new settings that allow customization of important strings without adding a language file in 96cd794. Hopefully that will make it easier to set the plugin up for most people who just need to change a couple of labels.

Thanks!

-David

**Comment by gdecuir777** [comment]
Hi David,

I tried and got the following error message

Parse error: syntax error, unexpected '/' in /home/myprayer/public_html/wp-content/plugins/hierarchical-groups-for-bp-master/hierarchical-groups-for-bp.php on line 90

**Comment by gdecuir777** [comment]
It brought my site down and only displays the error message

**Comment by dcavins** [comment]
@gdecuir777-

That's weird. Can you check that file to see if it looks like the file in the repo?
https://github.com/dcavins/hierarchical-groups-for-bp/blob/master/hierarchical-groups-for-bp.php

I don't see anything on line 90 that looks suspicious. What is in your version of that file?

Thanks!

-David

**Comment by gdecuir777** [comment]
Hi David,

Sorry my mistake. I didn't copy over all the files as I should have. It's working. I'll give you feedback shortly on how it works.

Thanks!

**Comment by gdecuir777** [comment]
Hi David,

I'm seeing the following for "child labels". I named my child label
"Subspaces". The number count is missing. Other then that all is working
fine.
- Subspaces <span class="count">1</span>
<http://myprayerspace.com/home/groups/journey-christian-church/hierarchy/>
-
- Thanks,
Gary

**Comment by dcavins** [comment]
Hi @gdecuir777-

You need to put a %s in your string where you want the number to appear, like `Subspaces %s`, and the count should be added. :)

-David

**Comment by gdecuir777** [comment]
Hi David,

I do have the %s in the string. Example "Subspaces %s". However, it's
showing Subspaces <span class="count">1</span>

Thanks,
Gary

**Comment by dcavins** [comment]
Ah, OK. I fixed that here:
https://github.com/dcavins/hierarchical-groups-for-bp/commit/0dc4832d70e45e36abf01343d05408dedbe757de

You must have gotten the ZIP before I committed some fixes and updates yesterday. There were a few: 
https://github.com/dcavins/hierarchical-groups-for-bp/commits/master

The current label change code is working OK on my sites. 

Thanks again for your help with testing!

-David
